### PR TITLE
Update README.adoc to describe our product

### PR DESCRIPTION
**Pull request by GabrielYik**
Take a look at the README.adoc and suggest any edits.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -15,26 +15,59 @@ ifndef::env-github[]
...
+.Current way of planning modules
+----
+1. Go to an external site to check what modules one has to take.
+2. See which modules' pre-requisites have been fulfilled and are open for one to take.
+3. Choose some of those modules.
+4. Start planning for the next semester.
+5. Repeat (1) - (4) for any number of semesters.
+6. Wishes to delete some modules
+7. Some modules planned now have their pre-requisites unfulfilled.
+8. Signs in frustration
```
Comment (rongjiecomputer) [inline README.adoc]: Signs -> Sighs

File: README.adoc [hunk 2]
```diff
@@ -15,26 +15,59 @@ ifndef::env-github[]
...
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== About
+
+This is a module planner application.
```
Comment (rongjiecomputer) [inline README.adoc]: Any reason why the list syntax (`- * blah blah`) was dropped? I think points can be presented clearer in list form.

File: README.adoc [hunk 3]
```diff
@@ -15,26 +15,59 @@ ifndef::env-github[]
...
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
 * <<ContactUs#, Contact Us>>
 
 == Acknowledgements
 
-* Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
-_Marco Jakob_.
-* Libraries used: https://github.com/TestFX/TestFX[TextFX], https://bitbucket.org/controlsfx/controlsfx/[ControlsFX], https://github.com/FasterXML/jackson[Jackson], https://github.com/google/guava[Guava], https://github.com/junit-team/junit5[JUnit5]
+* The source code that the module planner is built on is from the
```
Comment (rongjiecomputer) [inline README.adoc]: This part sounds a bit weird. Perhaps "The source code of this module planner is built on top of AddressBook-Level4 ..." sounds better?

File: README.adoc (hunk 1 above)
Comment (GabrielYik) [reply README.adoc]: Fixed

File: README.adoc (hunk 2 above)
Comment (GabrielYik) [reply README.adoc]: Ok reverted back to points. I didn't do the same for the "So tedious" and "Much better!" cos it seems kinda odd for them to be in points

Comment (rongjiecomputer) [reply README.adoc]: Current one is fine. Thanks!

**Pull request merged**